command/seal_migration_test.go

```diff
@@ -441,11 +419,6 @@ func testSealMigrationTransitToTestSeal(t *testing.T, setup teststorage.ClusterS
 		t.Fatal(err)
 	}
 
-	// Nuke the transit server; assign nil to Cores so the deferred Cleanup
```

Comment:
I think idea here was good even if the location was not.  What do you think about adding another section at the end of the current method: now that we've migrated we should be able to unseal without using the original transit server, so kill tcluster and go through another seal/unseal?

Reply:
Good idea, I pushed a commit that does that very thing.